src/hotspot/os/bsd/os_bsd.cpp

```diff
@@ -2133,23 +2133,32 @@ jint os::init_2(void) {
   FLAG_SET_ERGO(UseNUMAInterleaving, false);
 
   if (MaxFDLimit) {
-    // set the number of file descriptors to max. print out error
+    // Set the number of file descriptors to max. print out error
```

Comment:
You dropped the other editorial fixed, but only kept part of this one. In the previous
version of patch (8300088), you also fixed: s/print/Print/

I would drop this editorial fix also.

Reply:
Done.